app/src/components/layout/Footer.test.tsx

```diff
@@ -1,11 +1,11 @@
+import { render } from '@testing-library/react';
 import React from 'react';
-import renderer from 'react-test-renderer';
 import Footer from './Footer';
 
 describe('Footer', () => {
   it('renders correctly', () => {
-    const tree = renderer.create(<Footer />).toJSON();
+    const { container } = render(<Footer />);
 
-    expect(tree).toMatchSnapshot();
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
how is this different than:

```
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
```

Reply:
seems like only checking the firstchild could maybe be an issue in some components?

Reply:
ooooh, wait, asFargment is a function.  Didnt pick up on that. 

When I ran `expect(asFragment).toMatchSnapshot();` it just generated a snapshot with a `['function']` as the content, which seemed wrong